svd-encoder/src/lib.rs

```diff
@@ -78,6 +78,7 @@ mod fieldinfo;
 mod interrupt;
 mod modifiedwritevalues;
 mod peripheral;
+mod peripheralinfo;
```

Comment:
imo, this should be `peripheral_info` but since we're already using this format, it makes sense.